python/python/bystro/parent_of_origin/tests/test_parent_of_origin.py

```
@@ -244,12 +244,14 @@ def test_decision_function():
     beta_m = np.zeros(n_p)
     beta_p = np.zeros(n_p)
     beta_p[:3] = 0.5
-    data = generate_data(beta_m, beta_p, rng, maf=0.1, n_individuals=20000)
+    data = generate_data(beta_m, beta_p, rng, maf=0.1, n_individuals=200000)
     model = POESingleSNP(
         compute_pvalue=True, cov_regularization="QuadraticInverse"
     )
     model.fit(data["phenotypes"], data["genotype"])
     diff = beta_p - beta_m
+    X = data['phenotypes']
```

Ruff

Local variable `X` is assigned to but never used

(F841)